Resources/Prototypes/SS220/Entities/Clothing/Head/HotMasks.yml

```
@@ -0,0 +1,64 @@
+- type: entity 
+  parent: ClothingHeadBase
+  id: ClothingHeadHatAlex
+  name: Алекс #С большой, ибо оффовская маска Ричарда тоже
```

```suggestion
  name: маска Алекса
```
Это ж по идее тоже маска, может тоже поставить в начале "маска"? или так и надо?

---

Так и надо